.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a bug report to help us fix things
+title: "[codename] Issue Title"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Device (please complete the following information):**
+ - Watch Codename: [e.g. sturgeon]
+ - AsteroidOS Builddate [e.g. 2022-03-06]
```

Comment:
This may not always apply. This might depend on a future addition. We don't actually mention the build date.
(But that's a whole other issue)